### ISSUE TITLE
Tag proper releases, and include compiled binaries and installers

### ISSUE THREAD

**Issue by DeflateAwning** [FrameworkComputer/framework-system#14]
A compiled window binary release (perhaps even built automatically with CI) would be great!

**Comment by JohnAZoidberg** [comment]
Yep, once we distribute the windows driver, we'll distribute compiled binaries as well.
The CI builds binaries already, you can fetch them yourself if you want.

**Comment by DeflateAwning** [comment]
Why not proper releases with semantic versioning?

**Comment by JohnAZoidberg** [comment]
We will, but without a signed driver it's not very useful on Windows, yet.
I uploaded the v0.1.0 binaries: https://github.com/FrameworkComputer/framework-system/releases/tag/v0.1.0
But it's not a meaningful release yet, just a debug build of the first available code.

**Comment by DeflateAwning** [comment]
Consider tagging a release of the latest version with the latest changes

**Comment by Thulium-Drake** [comment]
I'd like to add to this request :-)

Even though there's no signed drivers yet, the current means on finding out whatever is latest is a pain when creating RPMs/DEBs. And 0.x.y releases are fine for now and still makes packaging a bit easier.

On the packaging front, do you plan to make official packages for the distro's supported by Framework for tools like this?

**Comment by DeflateAwning** [comment]
Making this installable from `cargo` (thus publishing it to crates.io) would be very handy too. Cargo isn't necessarily a package manager, but it does work shocking well for distributing software like this.

**Comment by Thulium-Drake** [comment]
@JohnAZoidberg Hi there! Any progress on getting the Windows drivers signed (as that's a prerequisite for upping the version number :-) ) ?

Thanks! 🚀

**Comment by JohnAZoidberg** [comment]
Yes, in fact there is progress! :D
We recently started getting the driver ready for release and the signing process.
But it's not an easy process, for one, we found out that our EV certificate that is okay for signing Windows executables, doesn't work for drivers, because Microsoft doesn't accept those with SHA384, only SHA256. So we got a new one and are trying again.

**Comment by Thulium-Drake** [comment]
Oof, well, good luck with that! :-)

**Comment by JohnAZoidberg** [comment]
We have been tagging releases and we have been including the EC windows driver in the official driver bundle.
